ci/quickstart-confluence-aurora.json

```diff
@@ -1,11 +1,11 @@
 [
   {
     "ParameterKey": "DBMasterUserPassword",
-    "ParameterValue": "$[taskcat_genpass_8S]"
+    "ParameterValue": "f925dO1ry_"
```

Comment:
Is this a workaround for the issue with `&` characters? Also, there isn't a build for this parameter set currently correct? Just checking 👍

Reply:
Yes and yes. In the next phase, we can add Bamboo variable for the param set that can be overridden either by a different job or manually via explicit execution.